ludwig/schema/utils.py

```diff
@@ -17,6 +17,8 @@
 from ludwig.schema.metadata.parameter_metadata import convert_metadata_to_json, ParameterMetadata
 from ludwig.utils.torch_utils import activations, initializer_registry
 
+RECURSION_STOP_ENUM = {"weights_initializer", "bias_initializer"}
```

Comment:
Let's add `norm_params` to this list as well.

Reply:
Good call!